README.md

```diff
@@ -10,7 +10,7 @@
 
 First, install all [build dependencies](docs/dev/dependencies.md).
 
-After cloning this repository, the installer binary will need to be built by running the following:
+After cloning this repository to your $GOPATH/src/github.com/openshift/installer directory, the installer binary will need to be built by running the following:
```

Comment:
`$GOPATH/...` doesn't work if the variable contains multiple paths (you can do that with [colon delimiters on Unix, and with other approaches on other OSes][1]).  I'd rather have something like:

```markdown
After cloning this repository to your [`GOPATH`][GOPATH], the installer binary will need to be built by running the following:
```
with the reference defined at the end of the file:

```markdown
[GOPATH]: https://golang.org/cmd/go/#hdr-GOPATH_environment_variable
```

For similar usage of "in the `GOPATH`" meaning "as a descendent of the paths" and not strictly "as a child of one of the paths", see [here][2] with:

> Vendor directories do not affect the placement of new repositories being checked out for the first time by 'go get': those are always placed in the main `GOPATH`, never in a vendor subtree.

While my suggestion is more generic, it's not as useful as what you currently have now for folks that happen to only have one path in their `GOPATH`.  But I think the error messages from #474 are enough to cover folks who misunderstand.

[1]: https://golang.org/cmd/go/#hdr-GOPATH_environment_variable
[2]: https://golang.org/cmd/go/#hdr-Vendor_Directories

Reply:
@wking 
Thx. I think the major issue in my test is about github.com/openshift/, but not $GOPATH. Quite agree with your above suggestion about GOPATH. But #474 seems not hint about installer/ need to be placed in github.com/openshift/ only if folks read it from build.sh.
When first try, I hit following error.
'''
./hack/build.sh 
+ LAUNCH_PATH=/root/work/installer
++ dirname ./hack/build.sh
+ cd ./hack/..
++ go list -e -f '{{.Dir}}' github.com/openshift/installer
+ PACKAGE_PATH=
+ test -z ''
++ go env GOPATH
+ echo 'build from your ${GOPATH} (/root/work/installer is not in /home/work/)'
build from your ${GOPATH} (/root/work/installer is not in /home/work/)
+ exit 1
'''
So I according the message to change my LAUNCH_PATH to my GOPATH. Still hit the issue.
'''
./hack/build.sh 
+ LAUNCH_PATH=/home/work/src/installer
++ dirname ./hack/build.sh
+ cd ./hack/..
++ go list -e -f '{{.Dir}}' github.com/openshift/installer
+ PACKAGE_PATH=
+ test -z ''
++ go env GOPATH
+ echo 'build from your ${GOPATH} (/home/work/src/installer is not in /home/work/)'
build from your ${GOPATH} (/home/work/src/installer is not in /home/work/)
+ exit 1
'''
When it comes to here, so confused about how to continue only if you get the important info(github.com/openshift/installer) from build.sh.

Reply:
> `/home/work/src/installer is not in /home/work/`

Yeah, that's a bit confusing.  Maybe we can teach `build.sh` to print better messages in this case?

Reply:
@wking 
I have a update according to your suggestion from doc side in this pr. Could u help review it again? For build.sh, I think we can update it in another pr.